source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -2759,25 +2760,17 @@ export class TextInput extends Input {
                     button.classList.remove("iconOnly");
                     button.classList.add("textOnly");
 
-                    if (this.inlineAction) {
-                        button.textContent = this.inlineAction.title ? this.inlineAction.title : "Title";
-                    }
-                    else {
-                        button.textContent = "Title";
-                    }
+                    button.textContent = this.inlineAction?.title ?? "Title";
                 }
 
                 icon.src = this.inlineAction.iconUrl;
+                icon.alt = this.inlineAction.title ?? "Title";
 
                 button.appendChild(icon);
-
-                if (this.inlineAction.title) {
-                    button.title = this.inlineAction.title;
-                }
             }
             else {
                 button.classList.add("textOnly");
-                button.textContent = this.inlineAction.title ? this.inlineAction.title : "Title";
```

Comment:
Let's please not use the ? and ?? operators. Support for those was introduced somewhat recently in TS, and not every host supports them. If we are going to use them, we at least need to check with Teams that it's not a problem for them.

Reply:
Done.